src/routes/Apps/App/TrainDialogs.tsx

```diff
@@ -142,7 +144,7 @@ const defaultTagFilter = (intl: InjectedIntl) => ({ key: -1, text: Util.formatMe
 interface ComponentState {
     columns: IRenderableColumn[]
     sortColumn: IRenderableColumn
-    history: Activity[]
+    activityHistory: Activity[]
```

Comment:
Glad to see this renamed. Never liked the original name as could be history of anything and browsers already have history.

What keeps it from being `activities`?

It would match what we do for other arrays like `actions`, `trainDialogs`, etc. I noticed the `history` variable in WebChat also and wondered if it was re-used from their code.

Reply:
Thought about that, although there is some value in the name "history" is that it represent the activities that existed before the new teach session started.  There could be new activities in the webchat